modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -505,6 +505,8 @@
 			</argument>
 			<description>
 				Returns [code]true[/code] if [code]a[/code] and [code]b[/code] are approximately equal to each other.
+				Here, approximately equal means that [code]a[/code] and [code]b[/code] are within a small, internal constant epsilon of each other.
```

Comment:
```suggestion
				Here, approximately equal means that [code]a[/code] and [code]b[/code] are within a small internal epsilon of each other, which scales with the magnitude of the numbers.
```

Reply:
It is the smallest difference detectable that scales, rather than the internal constant, right?
`core/math/math_defs.h:#define CMP_EPSILON 0.00001`